README.rst

```diff
@@ -117,6 +117,8 @@ instead of ``except (SomeError,):``.
 **B014**: Redundant exception types in ``except (Exception, TypeError):``.
 Write ``except Exception:``, which catches exactly the same exceptions.
 
+**B015**: Pointless comparison.
```

Comment:
```suggestion
**B015**: Pointless comparison. This comparison does nothing but wastes CPU instructions. Remove it.
```

Maybe just be a little more verbose here and give an action.

Reply:
In practice we're not worried about CPU instructions (especially in Python), but about missing asserts in pytest-style tests, so the wording is a bit strange here. How about "Did you forget an assert?" instead?